src/Console/PurgeCommand.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Laravel\Dusk\Console;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+
+class PurgeCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'dusk:purge';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Purge dusk test debugging files';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->ignoreValidationErrors();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->purgeScreenshots();
+
+        $this->purgeConsoleLogs();
+
+        $this->purgeSourceLogs();
+    }
+
+    /**
+     * Purge the failure screenshots.
+     *
+     * @return void
+     */
+    protected function purgeScreenshots()
+    {
+        $this->purgeDebuggingFiles(
+            'tests/Browser/screenshots', 'failure-*'
+        );
+    }
+
+    /**
+     * Purge the console logs.
+     *
+     * @return void
+     */
+    protected function purgeConsoleLogs()
+    {
+        $this->purgeDebuggingFiles(
+           'tests/Browser/console', '*.log'
+        );
+    }
+
+    /**
+     * Purge the source logs.
+     *
+     * @return void
+     */
+    protected function purgeSourceLogs()
+    {
+        $this->purgeDebuggingFiles(
+            'tests/Browser/source', '*.txt'
+        );
+    }
+
+    /**
+     * Purge debugging files based on path and patterns.
+     *
+     * @param  string  $relativePath
+     * @param  string  $patterns
+     * @return void
+     */
+    protected function purgeDebuggingFiles($relativePath, $patterns)
+    {
+        $path = base_path($relativePath);
+
+        if (! is_dir($path)) {
+            $this->warn(
+                "Ignored purging none existing [{$relativePath}] path.", OutputInterface::VERBOSITY_DEBUG
```

Comment:
Since English likes to play games with us, this is a homonym of the more grammatically-correct "non-existent".

The framework's `Filesystem::get()` method has a similar exception message so maybe the two can be combined:

https://github.com/laravel/framework/blob/b3bea36b9e3a5b6e5b1d12e2ddc259beff656982/src/Illuminate/Filesystem/Filesystem.php#L51-L58

> Purge skipped the directory that does not exist at [{$relativePath}] path.

Reply:
Updated, thanks @derekmd